pkg/tfbridge/schema.go

```diff
@@ -1407,7 +1406,12 @@ func extractInputs(oldInput, newState resource.PropertyValue, tfs shim.Schema, p
 }
 
 func getDefaultValue(tfs shim.Schema, ps *SchemaInfo) interface{} {
-	if dv, _ := tfs.DefaultValue(); dv != nil {
+	if dv, err := tfs.DefaultValue(); dv != nil {
```

Comment:
We have changed `DefaultValue` to return `nil, fmt.Errorf("default values not supported")`. The error won't even be logged here, since `dv == nil`. We need to also check that `err != nil`.